corehq/apps/es/filters.py

```diff
@@ -106,14 +92,14 @@ def doc_id(doc_id):
 def missing(field, exist=True, null=True):
     """Only return docs missing a value for ``field``"""
     return {
-        "missing": {
-            "field": field,
-            "existence": exist,
-            "null_value": null
-        }
+        "bool": {"must_not": exists(field)}
     }
 
 
+def field_exists(field):
+    return exists(field)
+
+
```

Comment:
nit: I don't think this is referenced anywhere anymore (and if it were, you could just use `exists` directly)
```suggestion
```

Reply:
Removed